src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,20 @@
+.stars {
+  display: flex;
+
+  &__star {
+    height: 16px;
+    width: 16px;
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    margin-right: 4px;
+    background-position: center;
+  }
+
+  &__star:last-child {
+    margin-bottom: 0;
```

Comment:
The `margin-bottom: 0;` property is not necessary here since the stars are displayed in a horizontal flexbox layout. Consider removing this line unless vertical spacing is needed in a different context.